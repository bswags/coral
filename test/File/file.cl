#cordbconn
server="";
user="";
password="memory";
port="";
DBName="";
type="sqlite";
#enddbconn

#cordb
Table User {
    name : string;
    age : int;
    primary_key(name);
 };
#enddb

void main() {

	File f;
	string b;
	connectDB;
	f = fopen("output", "w");
	fprintf(f, "hello");
	fprintf(f, c.name);
	b = freadline(f);
	
	while (b != "" ) {
        	printf(b);
        	b = freadline(f);
    	}

    	fclose(f);

	closeDB;

}
