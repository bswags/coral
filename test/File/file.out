Traceback (most recent call last):
  File "File/file.clx", line 6, in <module>
    setServer("")
NameError: name 'setServer' is not defined
